pkg/network/message_test.go

```diff
@@ -1 +1,32 @@
 package network
+
+import (
+	"testing"
+	"time"
+
+	"github.com/nspcc-dev/neo-go/pkg/io"
+	"github.com/nspcc-dev/neo-go/pkg/network/payload"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEncodeDecode(t *testing.T) {
```

Comment:
I think we need more tests to cover this changeset.

Reply:
I added test for boundary cases to check whether compression works correctly. Do we need tests for all message types?

Reply:
I think it's enough to cover changes made by this PR which is mostly done as of now.